core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -123,50 +123,36 @@ public String startWorkflow(String name, int version, String correlationId, Map<
 	}
 
 	public String startWorkflow(String name, int version, String correlationId, Map<String, Object> input, String event, Map<String, String> taskToDomain) throws Exception {
-		return startWorkflow(null, name, version, input, correlationId, null, null, event, taskToDomain, null, null, null);
+		return startWorkflow(null, name, version, input, correlationId, null, null, event, taskToDomain, null, null, null, null);
```

Comment:
We should consider refactoring these `startWorkflow()` methods at some point. Accepting this many parameters isn’t ideal.

Reply:
Totally agree! Just wanted to keep in the same way the Netflix designed it

Reply:
That makes sense. I haven't checked, but maybe they addressed this in their code in a later release.